h2o-bindings/bin/custom/R/gen_stackedensemble.py

```diff
@@ -56,6 +56,7 @@ def update_param(name, param):
 if (!missing(metalearner_params)) {
     model@parameters$metalearner_params <- list(fromJSON(model@parameters$metalearner_params))[[1]] #Need the `[[ ]]` to avoid a nested list
 }
+model@model$base_models <- unlist(base_models)
```

Comment:
see comments for Py API below.
`model@model$base_models` should be fed from backend value if available, and formatted as a vector of model ids, so that it can directly be reused (for another SE for example).